CesiumGltf/generated/include/CesiumGltf/ModelSpec.h

```diff
@@ -137,6 +137,54 @@ struct CESIUMGLTF_API ModelSpec : public CesiumUtility::ExtensibleObject {
    */
   std::vector<CesiumGltf::Texture> textures;
 
+  int64_t getSizeBytes() const {
```

Comment:
Can we add a generated comment to these getSizeBytes methods explaining what it does, and also mentioning that it requires a traversal of the entire structure, which may be slow?